_data/projects/ElectronClippy.yml

```diff
@@ -0,0 +1,10 @@
+name: Clippy
+desc: Electron App that records your Clipboard history and Image history
+site: https://github.com/Jarmahent/Clippy
+tags:
+- electron
+- javascript
+- react
+upforgrabs:
+  name: Help Needed
+  link: https://github.com/Jarmahent/Clippy
```

Comment:
@Jarmahent Please change this to https://github.com/Jarmahent/Clippy/labels/help%20wanted in case you want to use "help wanted" label. Here, help wanted is just an example.

Reply:
@Jarmahent Please change the link too. Also, please change the name to "help wanted" and mark some issues in your repository with "help wanted" label.